sections/futures/Trade/ManagePosition.tsx

```diff
@@ -56,8 +56,8 @@ const ManagePosition: React.FC<ManagePositionProps> = ({
 
 				<ManagePositionContainer>
 					<PlaceOrderButton
-						variant="primary"
-						fullWidth
+						noOutline={true}
```

Comment:
By the way, to save you a few keystrokes, you can do this instead:
```suggestion
						noOutline
```

Reply:
Thanks, will fix it!